workshops/forms.py

```diff
@@ -56,15 +57,43 @@ class InstructorsForm(forms.Form):
         lookup_class=lookups.AirportLookup,
         label='Airport',
         required=False,
-        help_text=AUTOCOMPLETE_HELP_TEXT,
+        widget=selectable.AutoComboboxSelectWidget(
+            lookup_class=lookups.AirportLookup,
+        )
```

Comment:
I have been following the issue tracker and have watched the video already. It was a useful git tutorial. Thanks :)